libs/src/index.js

```diff
@@ -174,7 +174,8 @@ class AudiusLibs {
     comstockConfig,
     captchaConfig,
     isServer,
-    isDebug = false
+    isDebug = false,
+    enableUserReplicaSetManagerContract = true
```

Comment:
this should probably be default turned off

Reply:
fixed